### ISSUE TITLE
ingress-nginx helm chart showing no resources

### ISSUE THREAD

**Issue by musabmasood** [argoproj/argo-cd#5821]
Everything else is working fine but for some reason nginx-ingress application is not showing any resources in the UI. The chart is already deployed in the cluster and live.

My application:

```
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: ingress-nginx-public
  finalizers:
    - resources-finalizer.argocd.argoproj.io
spec:
  project: default
  source:
    repoURL: <private-repo>
    targetRevision: HEAD
    path: ingress-nginx
    helm:
      releaseName: ingress-nginx-public
      valueFiles:
      - values-overrides.yaml
  destination:
    server: https://kubernetes.default.svc
    namespace: ingress-nginx
  syncPolicy: {}
```

The way I proxy the nginx-ingress helm chart:

```
# my nginx-ingress proxy helm chart
version: v2
name: ingress-nginx
version: 0.1.0
dependencies:
- name: ingress-nginx
  version: 3.24.0
  repository: https://kubernetes.github.io/ingress-nginx

```

On the UI it shows as healthy and synced but there are no resources being shown in the tree

![image](https://user-images.githubusercontent.com/8170066/111817850-e573e280-88b4-11eb-91a1-7408b792cbad.png)

Side notes:

- No errors on argocd server
- No errors on argocd UI
- Chart already deployed infact argocd is being served by this nginx-ingress controller
- ArgoCD version `1.8.7+eb3d1fb`

Any ideas what is wrong with this deployment?

**Comment by musabmasood** [comment]
Some more context. In my proxy chart that resides in my git repo, I added a `templates/test-pod.yaml` with a simple test pod. When I pushed this change, the test pod **did show up** as out of sync in the UI. However, none of the manifests from the dependent chart (ingress-nginx) are showing up.

This makes me believe that for some reason argocd is not running `helm dep up` and not generating the manifests for the dependent chart. Something like that.
